### DISCUSSION TITLE
CoastalApp code maintenance

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [noaa-ocs-modeling/CoastalApp#149] [General]
## WW3 and ADCIRC fork for OCS-Modeling for CoastalApp
@pvelissariou1 Please add your rationale here to get started!

**Comment by pvelissariou1** [comment]
Code maintainance of WW3/ADCIRC and other models should be the responsibility of the respective developers.
CostalApp submodules should point to the upstream repos and not to their forked versions.
We need a schedule for the updates of  CoastalApp submodules followed by the testing of the updated CoastalApp
(may be bi-weekly or monthly)

**Comment by pvelissariou1** [comment]
Tasks:

1. Merge develop-updates to develop and main
2. Delete the develop_build branch
